List-1/make_pi.py

```diff
@@ -0,0 +1,2 @@
+def make_pi():
+    return [3,1,4]
```

Comment:
thats not the question. `pi` is just a placeholder..

What if I want to display 3 digits of 100 divided by 3. 

You cannot hard code like this.